editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -104,7 +99,7 @@ HelpBrowser::HelpBrowser(QWidget* parent): QWidget(parent) {
 
     // Delete the help browser's page to avoid an assert/crash during shutdown. See QTBUG-56441, QTBUG-50160.
     // Note that putting this in the destructor doesn't work.
-    connect(QApplication::instance(), &QApplication::aboutToQuit, [webPage]() { delete webPage; });
+    connect(QApplication::instance(), &QApplication::aboutToQuit, [=]() { delete mWebView->page(); });
```

Comment:
i think it can just be this?

```suggestion
    connect(QApplication::instance(), &QApplication::aboutToQuit, [mWebView]() { delete mWebView->page(); });
```

Reply:
That gives me "error: capture of non-variable mWebView", as we can't capture data members by value. We can capture `this` though:
```
connect(QApplication::instance(), &QApplication::aboutToQuit, [this]() { delete mWebView->page(); });
```

Reply:
oops, thanks for catching that